### PR TITLE
docs: Describe support links

### PR DESCRIPTION
**Pull request by mtojek**
Related: https://github.com/coder/coder/pull/6313

This PR adds a doc page for support links.

### CODE REVIEW

File: docs/admin/support-links.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Support Links
+
+Support links let admins adjust the user dropdown menu to include links referring to internal company resources. The menu section replaces the original menu positions: documentation, report a bug to GitHub, or join the Discord server.
+
+![support links](../images/admin/support-links.png)
```
Comment (johnstcn) [inline docs/admin/support-links.md:5]: In this context, we should probably show the default support links.

File: docs/admin/support-links.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Custom links can be set in the deployment configuration using the `server.yaml` file:
+
+```yaml
+support:
+  links:
+    - name: "RTFM 🔥"
```
Comment (johnstcn) [inline docs/admin/support-links.md:12]: Perhaps too spicy for docs?

File: docs/admin/support-links.md (hunk 1 above)
Comment (mtojek) [reply docs/admin/support-links.md:5]: Fixed!

File: docs/admin/support-links.md (hunk 2 above)
Comment (mtojek) [reply docs/admin/support-links.md]: Rephrased 👍

**Review by kylecarbs — APPROVED** [review]

**Pull request merged**